### ISSUE TITLE
indent bug?

### ISSUE THREAD

**Issue by evilebottnawi** [eslint/eslint#4696]
Eslint v1.10.3 
```json
{
        "indent": [
            2,
            4,
            {
                "SwitchCase": 1
            }
        ]
}
```
```javascript
            $thumbnail
                .stop()
                .animate(
                    {
                        height: productThumbnailHeight
                    },
                    Math.abs(productThumbnailHeight - $thumbnail.height()) * 2,
                    function () {
                        $thumbnail.css({
                            height: 'auto',
                            overflow: ''
                        });
                    }
                );
```
```bash
  29:21  error  Expected indentation of 16 space characters but found 20  indent
  30:25  error  Expected indentation of 20 space characters but found 24  indent
  31:21  error  Expected indentation of 16 space characters but found 20  indent
```

**Comment by eslintbot** [comment]
@evilebottnawi Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by gyandeeps** [comment]
Can you please narrow down your error message wrt the code as I cant understand which is line 29?

**Comment by evilebottnawi** [comment]
@gyandeeps 
```javascript
$thumbnail
    .stop()
    .animate(
        {
            height: productThumbnailHeight
        },
        Math.abs(productThumbnailHeight - $thumbnail.height()) * 2,
        function () {
            $thumbnail.css({
                height: 'auto',
                overflow: ''
            });
        }
    );
```
```bash
4:9   error  Expected indentation of 4 space characters but found 8   indent
5:13  error  Expected indentation of 8 space characters but found 12  indent
6:9   error  Expected indentation of 4 space characters but found 8   indent
```

**Comment by gyandeeps** [comment]
I think this is a know issue, Let me see if I can find an existing issue for something similar.

**Comment by evilebottnawi** [comment]
@gyandeeps maybe add label bug? i don't have time now to search existing issue

**Comment by gyandeeps** [comment]
Related https://github.com/eslint/eslint/issues/4081
and/or https://github.com/eslint/eslint/issues/4076

**Comment by ilyavolodin** [comment]
@gyandeeps I think this one is slightly different than the issues you linked. This one is about multiline braces, and the one you linked is about multiline chaining.

**Comment by nzakas** [comment]
@ilyavolodin this one is about chaining too

**Comment by grawlinson** [comment]
I posted previously about [this](https://github.com/eslint/eslint/issues/3832#issuecomment-168596308) on a closed issue, not sure if anyone saw it.

**Comment by nzakas** [comment]
@grawlinson generally closed issues get very little attention, always best to find an open one or create a new one.

**Comment by grawlinson** [comment]
Upon further investigation it seems that the problem was within my editor (Atom) entering spaces instead of tabs. Once I set `tabType: "hard"` and changed all spaces -> tabs, my problems disappeared.

**Comment by unional** [comment]
I have the same issue on webStorm with stroustrup brace-style:

```
{
  rules: {
    'brace-style': [2, 'stroustrup'],
    indent: [2, 4, {'SwitchCase': 1, 'VariableDeclarator': 1}]
  }
}
```

code:
```js
var browseButtonConfig = Ext.apply({
        cls: 'plain-btn customize-logo-btn',  // indent error
        iconCls: 'icon-commit'  // indent error
    },  // indent error
    xyz
);

var browseButtonConfig = Ext.apply(
    {
        cls: 'plain-btn customize-logo-btn',  // no error
        iconCls: 'icon-commit'  // no error
    },  // no error
    xyz
);
```

**Comment by q42jaap** [comment]
I have the same question about this https://github.com/eslint/eslint/issues/5097

**Comment by kaicataldo** [comment]
Closing this because we haven't made any progress in months. Thanks for contributing!
